src/visualization.cpp

```diff
@@ -191,4 +191,93 @@ void visualize(const World& world, const std::vector<RobotPtr>& robots, bool col
     cv::waitKey(3);
 }
 
+visualization_msgs::MarkerArray create_rviz_objectmsg(const World &world, const MapConfig &mapconfig)
```

Comment:
This function visualises the map as well as the objects. question is whether this is how we want to do visualisation. As it means we cannot visualise on the robot the same way we do in simulator

Reply:
see below

Reply:
robots and walls are skipped in lines 229 and 231